GEON-PPANG-iOS/Presentation/Common/StackView/LinkButtonStackView.swift

```diff
@@ -64,20 +63,22 @@ final class LinkButtonStackView: UIStackView {
             $0.distribution = .equalSpacing
         }
         
-        // TODO: 클릭 시 사파리로 이동 (API 통신 필요)
-        // 이 부분 귀찮아서 이렇게 해버렸는데 더 좋은 방법이 있다면 인사이트 공유 부탁합니다 ..
         [homepageLinkButton, instagramLinkButton].enumerated().forEach { index, button in
+            let tappedLinkButton = index == 0 ? self.tappedHomepageLinkButton : self.tappedInstagramLinkButton
+            
             button.do {
                 $0.setTitle(labeling[index], for: .normal)
                 $0.setTitleColor(.gbbGray400, for: .normal)
                 $0.setUnderline()
                 $0.titleLabel?.font = .subHead
+                $0.addAction(UIAction { _ in
+                    tappedLinkButton?()
+                }, for: .touchUpInside)
                 $0.isHidden = true
             }
         }
```

Comment:
p3:
흠 여기서 enumerated() 를 쓰는건 좀... 어색한거같은데 어케 생각하세여 ㅎㅋㅎㅋ 차라리 이렇게는 어떨까여

```suggestion
        [homepageLinkButton, instagramLinkButton].forEach { button in
            button.do {
                $0.setTitle(labeling[index], for: .normal)
                $0.setTitleColor(.gbbGray400, for: .normal)
                $0.setUnderline()
                $0.titleLabel?.font = .subHead
                $0.isHidden = true
            }
        }
        
        homepageLinkButton.addAction(UIAction { [weak self] _ in
            self?.tappedHomepageLinkButton
        }, for: .touchUpInside)
        
        instagramLinkButton(UIAction { [weak self] _ in
            self?.tappedInstagramLinkButton
        }, for: .touchUpInside)
```

Reply:
아 이거 원래 있던 코드에서 수정 많이 하기 귀찮아서 ㅋㅎ ,,
바꿀게엽 근데 weak self는 왜 쓰는 걸까여 ??

Reply:
이거 걍 코드 너무 길어지고 애매해서 원래대로 스트링 처리만 하고 냅둘게여 ~
[8aca5d5](https://github.com/GEON-PPANG/GEON-PPANG-iOS/pull/188/commits/8aca5d540b4f4682f7b550571c84964fca0a67b9)